runtime/autoload/provider/pythonx.vim

```diff
@@ -9,19 +9,29 @@ let s:stderr = {}
 let s:job_opts = {'rpc': v:true}
 
 " TODO(bfredl): this logic is common and should be builtin
-function! s:job_opts.on_stderr(chan_id, data, event)
-  let stderr = get(s:stderr, a:chan_id, [''])
-  let last = remove(stderr, -1)
-  let a:data[0] = last.a:data[0]
-  call extend(stderr, a:data)
+function! s:job_opts.on_stderr(chan_id, data, event) dict
+  let stderr = get(s:stderr, a:chan_id, [])
```

Comment:
I found I couldn't get any error stack in my branch, but that no longer a case for latest branch, so I've revert the changes.